Mobile-Development-and-React-Native.md

```diff
@@ -91,8 +91,54 @@ The values that can be used are:
 -   `sentences`: first letter of each sentence (default)
 -   `none`: do not capitalize automatically
 
+## Optimization
+
+`Before run into the optimization rabbit role in your applications:`
+> Measure. Don’t tune for speed until you’ve measured, and even then don’t unless one part of the code overwhelms the rest. - [Rob Pyke, 5 Rules of Programming](http://users.ece.utexas.edu/~adnan/pike.html)
+
+### Images
+
+Allways try to use the smallest possible image size. The larger the file, the longer they will take to transfer and display on the screen. Use image resizing software to compress the image. Remember to find a good balance between the quality and size of the photo.
+
+`Note: some image pickers has the built-in option to apply some compress or resize to the image.`
```

Comment:
Just some english nitpicks :stuck_out_tongue:
- `*Always* try to use (...)`
- I think that it would read better if it was `(...) The larger the file, the longer *it* will take to transfer(...)`
- `Note: some image pickers *have* the built-in option (...)`